products/firefox_iOS/testing_iOS.md

```diff
@@ -0,0 +1,22 @@
+# Testing on iOS for a new release
+We currently rely on screenshots for testing. These are provided on the [dev.l10n mailing list](https://lists.mozilla.org/listinfo/dev-l10n), which you should follow if you are working on l10n for any of the existing Mozilla products (otherwise you are missing crucial information and updates).
+
+## Testing with screenshots
+Screenshots are currently provided by the iOS team on dev.l10n, and the usual link where you can check out the latest versions can be found [right here](https://people-mozilla.org/~sarentz/fxios/screenshots/).
+
+## Test builds
+These are currently not set-up anymore - we are working with the iOS team to get them going again. Stay tuned!
```

Comment:
~~we~~
Possible alternative:

> Currently testing builds are not available. The iOS team is working on creating a new channel with builds dedicated to testing and open to localizers.

BTW, aren't Test Flight builds still available? I still get the notifications